components/common/autoTeleport/AutoTeleportActionButton.vue

```diff
@@ -191,6 +191,15 @@ const hasNoFundsAtAll = computed(
   () => !hasEnoughInCurrentChain.value && !hasEnoughInRichestChain.value,
 )
 
+const confirmButtonTitle = computed(() => {
+  const interaction =
+    transactions.value.actions[0].interaction?.toLocaleLowerCase()
+  return (
+    $i18n.t(`autoTeleport.steps.${interaction}.confirm`) ||
```

Comment:
I just want to confirm. Is it safe to return this? Afaik `$i18n.t` always returns `true(string)`, so it won't return `autoTeleport.steps.common.confirm.`

Reply:
Good point. It does return the original key (string). It looks like we do not have a simple way to set the fallback value for translation. I have removed the `autoTeleport.steps.common.confirm.`